numpy/distutils/command/build_ext.py

```diff
@@ -271,6 +274,21 @@ def run(self):
                 os.makedirs(runtime_lib_dir)
             runtime_lib = os.path.join(self.extra_dll_dir, fn)
             copy_file(runtime_lib, runtime_lib_dir)
+            
+        # If the compiler is MSVC, then copy the msvcp DLL
+        # See scipy/scipy#7969 for more information
+        # Copy in an odd manner to avoid permission errors
+        if self.compiler.compiler_type == 'msvc' and need_cxx_compiler \
+                and sys.version_info >= (3, 5):
+            if not os.path.isdir(runtime_lib_dir):
+                os.makedirs(runtime_lib_dir)
+            msvcp_dll = self._get_msvcp_dll()
+            dst = os.path.join(runtime_lib_dir,
+                               os.path.basename(msvcp_dll))
+            print('copying ' + msvcp_dll + ' -> ' + dst)
+            with open(msvcp_dll, 'rb') as fsrc:
+                with open(dst, 'wb+') as fdst:
+                    shutil.copyfileobj(fsrc, fdst)
```

Comment:
Nope. The comment explains it:

># Copy in an odd manner to avoid permission errors

`shutil.copyfile` may use the winapi to copy the file, which may copy the permissions as well. The permissions on that particular file make it read only; the `wheel` plugin cannot move it later when it wants.